src/providers/twitch/TwitchChannel.cpp

```diff
@@ -135,10 +111,6 @@ TwitchChannel::TwitchChannel(const QString &name)
         }
     });
 
-    // timers
-    QObject::connect(&this->chattersListTimer_, &QTimer::timeout, [=] {
-        this->refreshChatters();
-    });
```

Comment:
Why do we want to disable this timer?

Reply:
The chatter list can only be refreshed if you are a moderator now and it is automatically refreshed if a mod uses the /chatters command. So the only reason to update it is so that mods would have an updated @ list. I don't mind adding it back but I didn't know if it is worth it at that point.